third_party/gtest/gtest_mpi_listener.cpp

```diff
@@ -0,0 +1,387 @@
+/*******************************<GINKGO LICENSE>******************************
```

Comment:
Should we be adding the Ginkgo license to this file? Perhaps we should add an exception?